packages/wonder-blocks-core/util/types.js

```diff
@@ -91,7 +91,8 @@ type IdRef = string;
 type IdRefList = IdRef | Array<IdRef>;
 
 // Source: https://www.w3.org/WAI/PF/aria-1.1/states_and_properties
-// TODO(kevinb): figure out how to type ID ref lists
+// TODO(kevinb): convert to disjoint union of exact object types keyed on role
```

Comment:
I'm not sure I follow this - meaning that ensuring that each role only allows certain sets of aria props?